kubernetes/conf.yaml.example

```diff
@@ -2,14 +2,16 @@ init_config:
 
 instances:
   # The kubernetes check retrieves metrics from cadvisor running under kubelet on each node.
-  # It also queries kubelet for its health and the list of local-running pods.
+  # It also queries kubelet for its health and the list of local-running pods, and (optionally)
+  # the apiserver for services and events
   #
   # By default we assume we're running under docker and
   # that the kubelet read-only port with no auth is enabled.
   #
   # In this case we will use the address of the default router to
   # reach the kubelet and cadvisor APIs unless the environment variable
-  # KUBERNETES_KUBELET_HOST is found (if your node names can be resolved by pods, we recommend you set this variable to spec.nodeName through the downward API).
+  # KUBERNETES_KUBELET_HOST is found (if your node names can be resolved by pods, we recommend
+  # you set this variable to spec.nodeName through the downward API).
```

Comment:
maybe mention `hostIP` as well? https://github.com/kubernetes/kubernetes/pull/42717 This will make it much simpler.